src/controls/first_person.rs

```diff
@@ -1,11 +1,33 @@
 use cgmath;
 use mint;
-use std;
 
 use cgmath::Rotation3;
 use input::{Button, Input, Key};
 use object::Object;
 
+#[derive(Clone)]
+struct Keys {
+    pub forward: Option<Button>,
```

Comment:
any reason this can't use `KeyAxis`? it's not like you are going to map the forward/backward (for example) to mouse buttons anyway

Reply:
good point, I'll change this to `KeyAxis`

Reply:
What naming should I use?